package.json

```diff
@@ -26,6 +26,7 @@
     "url": "https://github.com/EOSIO/eosjs.git"
   },
   "dependencies": {
+    "bn.js": "^5.1.2",
```

Comment:
One more thing, we intentionally pin all dependencies in package.json, can you remove the `^` here?

Reply:
Done